packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -418,11 +418,16 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                                     // In order to decrease the number of times we have to execute a snapshot refresh,
                                     // if this is the summarizer and we have a cache entry but it is past the defaultSummarizerCacheExpiryTimeout,
                                     // force the network retrieval instead as there might be a more recent snapshot available.
-                                    if (this.hostPolicy.summarizerClient &&
-                                        age > defaultSummarizerCacheExpiryTimeout) {
-                                        // eslint-disable-next-line @typescript-eslint/dot-notation
-                                        props["cacheSummarizerExpired"] = true;
-                                        return undefined;
+                                    // See: https://github.com/microsoft/FluidFramework/issues/8995 for additional information.
+                                    if (this.hostPolicy.summarizerClient) {
+                                        if (age > defaultSummarizerCacheExpiryTimeout) {
+                                            // eslint-disable-next-line @typescript-eslint/dot-notation
+                                            props["cacheSummarizerExpired"] = true;
```

Comment:
We should also log the timeout value. If we change it later, it will be helpful to know the timeout to filter out logs.

Reply:
Instead of recording true/false, I'd rather record actual age (maybe always) and cutoff (60 seconds). That info is enough to do any kind of analyzes.

Reply:
We sort of do it for one path with cacheEntryAge.

Reply:
Yes we already record the age - the difference is whether it is a summarizer running it or not. I chose the Boolean because it would be easier to query…